python/lsst/meas/extensions/scarlet/io.py

```diff
@@ -153,13 +153,18 @@ def updateCatalogFootprints(
             # no models for its sources.
             continue
 
+        parent = catalog.find(parentId)
+        if updateFluxColumns and imageForResdistibution is not None:
```

Comment:
Is it worth fixing this to `imageForRedistribution` now?

Reply:
Sure, although I'll have to update a lot more packages now...